src/commands/publish/upload_form/mod.rs

```diff
@@ -63,16 +66,32 @@ pub fn build_script_upload_form(target: &Target) -> Result<Form, failure::Error>
                 let path = bundle.wasm_path();
                 let binding = bundle.get_wasm_binding();
                 let wasm_module = WasmModule::new(path, binding)?;
-                wasm_modules.push(wasm_module)
+                wasm_modules.push(wasm_module);
             }
 
-            let assets = ProjectAssets::new(script_path, wasm_modules, kv_namespaces)?;
+            let mut text_blobs = Vec::new();
+
+            if let Some(site) = &target.site {
+                let binding = "__STATIC_CONTENT_MANFIEST".to_string();
```

Comment:
I prefer making this a const, if possible :)

Reply:
calls in constants are limited to constant functions, tuple structs and tuple variants

Reply:
made with <3 from the compiler